ecl/hql/hqlfilter.cpp

```diff
@@ -1902,40 +1904,42 @@ IValueSet * FilterExtractor::createValueSetCompareExpr(IHqlExpression * selector
     if (!createConstantField(compareValue, selector, normalized))
         throwTooComplex(expr);
 
+    //TBD: Support substring matches
```

Comment:
Is this comment still valid?

Reply:
Yes - I should open a jira.  It will be needed once this code is used for filtering in esp.

Reply:
HPCC-22017